### DISCUSSION TITLE
Office Dialog API messageParent Mac.

### DISCUSSION THREAD

**Discussion by asfaarahmed** [OfficeDev/office-js#4130] [Q&A]
This is how i am opening dialog as it is recommended to open dialog from same origin.

`const showDialog = () => {

    Office.context.ui.displayDialogAsync(
      "https://localhost:3000/dialog.html",
      { height: 30, width: 20 },
      function (asyncResult) {
        const dialog = asyncResult.value;
        dialog.addEventHandler(Office.EventType.DialogMessageReceived, (arg) => {
          const res = JSON.parse(arg.message);
          if (res.success) {
            //login in success...
           } else {
            //noting
          }
        });
      }
    );
};`
    
On dialog.html i just redirected to
`Office.initialize = () => {
    window.location.href = "https://someWeb.com/testsample.html";  
};`

On my `someWeb.com/testsample.html` which is working on windows but not on Mac.
`Office.context.ui.messageParent("Msg", { targetOrigin: "*" });`
tried 
`Office.context.ui.messageParent(jsonMessage, { targetOrigin: "https://localhost:3000" });`
`Office.context.ui.messageParent("Msg");`
but no success on mac..

**Comment by ravivarma-highspot** [comment]
Any fix found for this issue? Even i am facing the similar issue now.
